backend/src/controllers/tasks.controller.ts

```diff
@@ -13,9 +21,25 @@ export class TasksController {
 
   @Post()
   insertTask(@Body() task: Task): Promise<void> {
+    if (task.effort < 0) {
```

Comment:
Can we move the validation to a new method to be reused in both the `insert` and the `update` methods?